docs/sdk/tutorials/importing_multilayer_geosat_assets.md

```diff
@@ -0,0 +1,167 @@
+<!-- FILE AUTO GENERATED BY docs/utils.py DO NOT EDIT DIRECTLY -->
+
+<a href="https://colab.research.google.com/github/kili-technology/kili-python-sdk/blob/main/recipes/importing_pdf_assets.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+
+# How to import multi-layer geosat assets to a Kili project
+
+In this tutorial, we will learn how to import multi-layer geosat assets to your project.
+
+Here are the steps that we will follow:
+
+1. Setting up an image project.
+2. Importing some multi-layer assets to Kili with GEOTIFFs and layers from a public tile server.
+
+## Setting up a Kili image project to work with
+
+### Installing and instantiating Kili
+
+First, let's install and import the required modules.
+
+```python
+%pip install kili
+```
+
+```python
+from pprint import pprint
+
+from kili.client import Kili
+```
+
+Now, let's set up variables needed to create an instance of the Kili object.
+
+We will need your API key and Kili's API endpoint.
+
+If you are unsure how to look up your API key, refer to [https://docs.kili-technology.com/docs/creating-an-api-key](https://docs.kili-technology.com/docs/creating-an-api-key).
+
+```python
+api_key = "YOUR_API_KEY"
+api_endpoint = f'https://cloud.kili-technology.com/api/label/v2/graphql'
+kili = Kili(api_endpoint=api_endpoint, api_key=api_key, verify=True)
+```
+
+### Creating an image Kili project
+
+To create an IMAGE Kili project, you must first set up its ontology.
+
+Here, we will only add a classification task:
+
+```python
+interface = {
+    "jobs": {
+        "OBJECT_DETECTION_JOB": {
+        "content": {
+            "categories": {
+            "PLANE": {
+                "children": [],
+                "color": "#472CED",
+                "name": "Plane",
+                "id": "category3"
+            }
+            },
+            "input": "radio"
+        },
+        "instruction": "BBOX",
+        "mlTask": "OBJECT_DETECTION",
+        "required": 0,
+        "tools": [
+            "rectangle"
+        ],
+        "isChild": False,
+        "isNew": False,
+        }
+  }
+}
+
+project = kili.create_project(
+    title="[Kili SDK Notebook]: Importing multi-layer Geosatellite asset",
+    description="Project Description",
+    input_type="IMAGE",
+    json_interface=interface,
+)
+```
+
+## Importing your assets
+
+### Download geotiff file examples
+
+Before adding assets you need to download our geotiff examples and add them in a geosat folder, created in the same folder as when you run your python script. This is mandatory to use local files and not url for geosatellite files.
```

Comment:
`created in the same folder as where you run your python script`
`It is mandatory to use local files and not url for geosatellite files` or `This step is mandatory to use...` depending on what you wanna say